man/avocado.rst

```diff
@@ -304,79 +306,103 @@ Options for subcommand `diff` (`avocado diff --help`)::
                             Comma separated filter of diff sections:
                             (no)cmdline,(no)time,(no)variants,(no)results,
                             (no)config,(no)sysinfo (defaults to all enabled).
-      --create-reports      Create temporary files with job reports (to be used by
-                            other diff tools)
+      --diff-strip-id       Strip the "id" from "id-name;variant" when comparing
+                            test results.
+      --create-reports      Create temporary files with job reports to be used by
+                            other diff tools
+
+    By default, a textual diff report is generated in the standard output.
 
 Options for subcommand `distro` (`avocado distro --help`)::
 
     optional arguments:
       -h, --help            show this help message and exit
       --distro-def-create   Creates a distro definition file based on the path
-                            given
+                            given.
```

Comment:
This is the only place you added the period. As most places do not have a period, I would follow the pattern and let this line without it.